app/helpers/alchemy/pages_helper.rb

```diff
@@ -81,7 +81,11 @@ def render_site_layout
     # @param [String] - Name of the menu
     # @param [Hash] - A set of options available in your menu partials
     def render_menu(name, options = {})
-      root_node = Alchemy::Node.roots.find_by(name: name, site: Alchemy::Site.current)
+      root_node = Alchemy::Node.roots.find_by(
+        name: name,
+        site: Alchemy::Site.current,
+        language: Alchemy::Language.current
```

Comment:
Since languages belong to sites we can remove the site_id query then.

```suggestion
        language: Alchemy::Language.current
```